newsroom/topics/topics.py

```diff
@@ -22,7 +22,7 @@ class TopicsResource(newsroom.Resource):
             'schema': newsroom.Resource.rel('users', required=True),
         },
         'timezone_offset': {'type': 'integer', 'nullable': True},
-        'topic_type': {'type': 'string', 'nullable': True},
+        'topic_type': {'type': 'string', 'required': True},
```

Comment:
I think setting an `allowed` list here would help validation, so the API client doesn't create an unsupported topic of type.
Allowed values should be `wire` or `agenda`